src/compas_ui/scene.py

```diff
@@ -74,15 +75,19 @@ def state(self):
         data = {}
         objects = []
         for obj in self.objects:
-            guid = str(obj.item.guid)
-            if guid not in data:
-                data[guid] = obj.item
+            item_guid = str(obj.item.guid)
+            guid = str(obj.guid)
+            parent_guid = str(obj.parent.guid) if obj.parent else None
+            if item_guid not in data:
+                data[item_guid] = obj.item
             objects.append({
-                'item': guid,
+                'guid': guid,
+                'item': item_guid,
                 'stream_id': obj.stream_id,
```

Comment:
shall we perhaps remove the stream id for now?

Reply:
indeed, removed all stream_id related code now